.github/workflows/haskell.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     runs-on: ${{ matrix.runner }}
     continue-on-error: ${{ matrix.ignore_error }}
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
You can revert this, the 9.0.2 CI ignores failure now. We need to fix that though.